Makefile

```diff
@@ -462,6 +462,7 @@ start-e2e-local: mod-vendor-local dep-ui-local cli-local
 	ARGOCD_APPLICATIONSET_CONTROLLER_NAMESPACES=argocd-e2e-external \
 	ARGOCD_APPLICATIONSET_CONTROLLER_ALLOWED_SCM_PROVIDERS=http://127.0.0.1:8341,http://127.0.0.1:8342,http://127.0.0.1:8343,http://127.0.0.1:8344 \
 	ARGOCD_E2E_TEST=true \
+	WORKQUEUE_FAILURE_COOLDOWN_NS=0 \
```

Comment:
As discussed, WORKQUEUE_FAILURE_COOLDOWN_NS=0  is the default and shouldn't be necessary to be added e2e. It would also serve as one proofpoint that existing behavior is preserved.

Reply:
Updated